test/unit/specs/components/Personalization/createViewCacheManager.spec.js

```diff
@@ -61,6 +62,10 @@ describe("Personalization::createCacheManager", () => {
     viewCacheManager.storeViews(decisionsDeferred.promise);
     decisionsDeferred.reject();
 
-    expectAsync(viewCacheManager.getView("cart")).toBeResolvedTo(undefined);
+    return expectAsync(viewCacheManager.getView("cart"))
+      .toBeResolvedTo(undefined)
+      .then(() => {
+        expect(viewCacheManager.isInitialized()).toBeTrue();
```

Comment:
I'm not sure if this should be true or false.

Consequences of true:
* if the initial request is canceled via the onBeforeRequestSend callback, future requests will not try to get personalization info (bad?)
* if the initial request fails due to Konductor/Network error, future requests will not try to get personalization info (good?)